packages/ra-ui-materialui/src/list/ListView.tsx

```diff
@@ -39,6 +39,7 @@ export const ListView = <RecordType extends RaRecord = any>(
     const {
         defaultTitle,
         data,
+        exporter,
```

Comment:
I think the change should be in ListActions, which should read the `exporter` from the `ListContext` rather than the props.